internal/controllers/clusterextension_controller_test.go

```diff
@@ -214,7 +227,10 @@ func TestClusterExtensionChannelVersionExists(t *testing.T) {
 	t.Log("By running reconcile")
 	res, err := reconciler.Reconcile(ctx, ctrl.Request{NamespacedName: extKey})
 	require.Equal(t, ctrl.Result{}, res)
-	require.NoError(t, err)
+	// Asserting that only loadError occurred
+	if assert.Error(t, err) {
```

Comment:
What's the value of returning an error aggregate? We can discuss in our sync on the error types.